ghost/email-service/lib/batch-sending-service.js

```diff
@@ -316,7 +343,13 @@ class BatchSendingService {
         let succeeded = false;
 
         try {
-            const members = await this.getBatchMembers(batch.id);
+            const members = await this.retryDb(
+                async () => {
+                    return await this.getBatchMembers(batch.id);
```

Comment:
We also need an empty rows check here. Which is a weird database issue that somehow isn't caught and is just returning 0 rows